emulate/emulate.js

```diff
@@ -4,6 +4,8 @@ const {NodeVM} = require("vm2");
 
 function generateEmulatorCode(userCode) {
   return `
+var request = require('request');
```

Comment:
Tego chyba nie musisz dopisywać, może to zrobić użytkownik

Reply:
oke